core/src/main/scala/org/apache/spark/util/SizeEstimator.scala

```diff
@@ -126,8 +127,8 @@ object SizeEstimator extends Logging {
   private def getIsCompressedOops: Boolean = {
     // This is only used by tests to override the detection of compressed oops. The test
```

Comment:
Given this comment I wonder if changing this one config is needed. (I also don't see any `SparkConf` available in this context, so looks that even that route is not possible.)

Reply:
there are cases when this config is set in the tests. It seems it is useless indeed, since we read it only here as a system property. I think we can consider removing the useless set as part of another PR related to that.

Otherwise we can try and replace the config with a string `TEST_USE_COMPRESSED_OOPS_KEY` which we use when setting/reading the system property. WDYT?

Reply:
the point is: it is set in `SparkConf` in some tests. But this is probably useless because we read it only here as a system property.

Reply:
I see; then it can probably be just removed from those tests.